bin/sync_plugins_to_cms.js

```diff
@@ -0,0 +1,126 @@
+// eslint-env node
```

Comment:
I originally started this in TypeScript (TS) with `.ts` files, but soon realized when I was writing tests, that although not that difficult to set up AVA for TS, enabling it caused existing tests to complain, so I took the middle road using types in JSDoc comments but still using the `tsx` package to get some stuff like top-level await.

Reply:
Fair fair, appreciate your jsdoc types nontheless!